contrib/babelfishpg_tds/src/backend/tds/guc.c

```diff
@@ -127,51 +128,57 @@ check_version_number(char **newval, void **extra, GucSource source)
 	len = strlen(*newval);
 	copy_version_number = palloc0(len + 1);
 	memcpy(copy_version_number, *newval, len);
+	current_version = (char *) GetConfigOption("babelfishpg_tds.product_version", true, false);
 	for (token = strtok(copy_version_number, "."); token; token = strtok(NULL, "."))
 	{	
 		/* check each token contains only digits */
 		if(strspn(token, "0123456789") != strlen(token))
 		{
-			ereport(ERROR,
-				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
-				errmsg("Please enter 4 valid numbers separated by \'.\' ")));
+			ereport(WARNING,
+				(errmsg("Please enter 4 valid numbers separated by \'.\' ")));
```

Comment:
If customers use a parameter group, they will notice this warning message later. So we should provide more details, e.g., `babelfishpg_tds.product_version cannot be set. Please enter ...`